packages/app/src/systems/Pool/pages/RemoveLiquidity.tsx

```diff
@@ -15,19 +15,31 @@ import {
   ZERO,
   TOKENS,
   useBalances,
+  useEthBalance,
 } from "~/systems/Core";
+import { useTransactionCost } from "~/systems/Core/hooks/useTransactionCost";
+import { getOverrides } from "~/systems/Core/utils/gas";
 import { Button, Card } from "~/systems/UI";
 
 export function RemoveLiquidityPage() {
   const navigate = useNavigate();
   const [errors, setErrors] = useState<string[]>([]);
   const contract = useContract()!;
   const balances = useBalances();
+  const ethBalance = useEthBalance();
 
   const liquidityToken = TOKENS.find((c) => c.assetId === CONTRACT_ID);
   const tokenInput = useCoinInput({ coin: liquidityToken });
   const amount = tokenInput.amount;
 
+  const txCost = useTransactionCost(["RemoveLiquidity-networkFee"], () =>
+    contract.prepareCall.remove_liquidity(1, 1, DEADLINE, {
```

Comment:
like we have in `Swap/utils/queries` this could be in a `Pool/utils/queries` as well